Packs/Akamai_WAF/ReleaseNotes/2_2_0.md

```diff
@@ -0,0 +1,19 @@
+
+#### Integrations
+
+##### Akamai WAF
+
+- Added support for **akamai-list-datastream-groups** command that returns access groups with contracts on your account. you can later use the groupid and contractid values to create and view streams or list properties by group. set the contractid query parameter to get groups for a specific contract.
+- Added support for **akamai-get-client_lists** command that get accessible client lists.
+- Added support for **akamai-patch-datastream** command that updates selected details of an existing stream. running this operation using json patch syntax creates a stream version that replaces the current one. currently you can patch a stream using only the replace operation. when updating configuration objects such as destination or deliveryconfiguration, pass a complete object to avoid overwriting current details with default values for omitted members such as tags, uploadfileprefix, and uploadfilesuffix. note that only active streams collect and send logs to their destinations. you need to set the activate parameter to true while patching active streams, and optionally for inactive streams if you want to activate them upon request. see patching streams for details.
+- Added support for **akamai-delete-datastream** command that deletes a deactivated stream. deleting a stream means that you can't activate this stream again, and that you stop receiving logs for the properties that this stream monitors. before deleting any stream, you need to deactivate it first. see deactivate a stream.
+- Added support for **akamai-list-datastreams** command that returns the latest versions of the stream configurations for all groups within the account.
+- Added support for **akamai-generic-api-call-command** command that akamai generic api call.
+- Added support for **akamai-list-idam-properties** command that lists the properties and includes for the current account.
+- Added support for **akamai-list-datastream-properties-bygroup** command that get properties that are active on the production and staging network and available within a specific group. run this operation to get and store the propertyid values for the create a stream and edit a stream operations.
```

Comment:
Added support for **akamai-list-datastream-properties-bygroup** command that gets properties active on the production and staging networks and available within a specific group. Run this operation to get and store the propertyid values for creating a stream and editing stream operations.